<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveMergedFilesDialog</class>
 <widget class="QDialog" name="SaveMergedFilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Concatenated Files Session As...</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioButtonNEV">
        <property name="text">
         <string>NEV</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupExtension</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonMAT">
        <property name="text">
         <string>MAT</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupExtension</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonNEX">
        <property name="text">
         <string>NEX</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupExtension</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonCSV">
        <property name="text">
         <string>CSV</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupExtension</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxOneOutputFile">
     <property name="text">
      <string>Save all sources into one destination file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxUseBaseFileName">
        <property name="text">
         <string>Use filename base</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditBaseFileName">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 159, 227, 10);
</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathLineEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 159, 227, 10);
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browsePB">
        <property name="toolTip">
         <string>Browse</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="OFSResources.qrc">
          <normaloff>:/new/Bitmaps/Bitmaps/backward18.png</normaloff>:/new/Bitmaps/Bitmaps/backward18.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>8</width>
          <height>8</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="filesTable">
     <property name="styleSheet">
      <string notr="true">alternate-background-color:  rgba(0, 159, 227, 10);

</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>5</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Source</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Destination</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="OFSResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SaveMergedFilesDialog</receiver>
   <slot>slotCheckInputs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SaveMergedFilesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotCheckInputs()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroupExtension"/>
 </buttongroups>
</ui>
